src/debug.c

```diff
@@ -482,9 +482,10 @@ void debugCommand(client *c) {
 "    Show low level client eviction pools info (maxmemory-clients).",
 "PAUSE-CRON <0|1>",
 "    Stop periodic cron job processing.",
-"REPLYBUFFER-PEAK-RESET-TIME <NEVER|time>",
+"REPLYBUFFER-PEAK-RESET-TIME <NEVER||RESET|time>",
```

Comment:
FYI, when m said "reset" i thought that the test suite will just set it back to 5.. but this way is ok too.

Reply:
I think it is better to keep defaults located at a single place so that in case we will decide to change it in the future will not have to cover many places...

Reply:
mainly checking that the value is between 1kib to 2kib after shrinkage